public/components/model_list/__tests__/model_list_filter.test.tsx

```diff
@@ -4,28 +4,55 @@
  */
 
 import React from 'react';
+import userEvent from '@testing-library/user-event';
 
 import { ModelListFilter } from '../model_list_filter';
 import { render, screen } from '../../../../test/test_utils';
 
 describe('<ModelListFilter />', () => {
-  it('should render default search bar with tag, stage and owner filter', () => {
-    render(<ModelListFilter value={{ tag: [], stage: [], owner: [] }} onChange={() => {}} />);
+  it('should render default search bar with tag, deployed and owner filter', () => {
+    render(<ModelListFilter value={{ tag: [], owner: [] }} onChange={() => {}} />);
     expect(screen.getByPlaceholderText('Search by name, person, or keyword')).toBeInTheDocument();
     expect(screen.getByText('Tags')).toBeInTheDocument();
-    expect(screen.getByText('Stage')).toBeInTheDocument();
     expect(screen.getByText('Owner')).toBeInTheDocument();
+    expect(screen.getByText('Deployed')).toBeInTheDocument();
+    expect(screen.getByText('Undeployed')).toBeInTheDocument();
   });
 
   it('should render default search value and filter value', () => {
     render(
       <ModelListFilter
         defaultSearch="foo"
-        value={{ tag: ['tag1'], stage: ['stage1'], owner: ['owner1'] }}
+        value={{ tag: ['tag1'], owner: ['owner1'] }}
         onChange={() => {}}
       />
     );
     expect(screen.getByDisplayValue('foo')).toBeInTheDocument();
-    expect(screen.queryAllByText('1')).toHaveLength(3);
+    expect(screen.queryAllByText('1')).toHaveLength(2);
```

Comment:
what's this `1`? shall we use a meaningful placeholder?

Reply:
The 1 for selected options for specific combobox, we may change to find with combobox name together.